### ISSUE TITLE
Make it possible to express independence within a layers contract

### ISSUE THREAD

**Issue by seddonym** [seddonym/import-linter#157]
It would be nice to express independent layers within a layers contract by putting the layers on one line, like so:

```
[importlinter:contract:my-layers-contract]
name = My three-tier layers contract (multiple root packages)
type = layers
layers=
    high
    medium_green medium_blue medium_yellow
    low
```

**Comment by flaeppe** [comment]
I've actually implemented a version of this in one of the projects I'm working on. With nearly identical config as you're suggesting here.

Let me see if I can port it to a PR draft here, if you want?

**Comment by flaeppe** [comment]
I pushed #162, if it's of interest. Have a look if you wish

**Comment by seddonym** [comment]
Thanks for the contribution! Will definitely look at the draft PR when I come to tackle this.

**Comment by seddonym** [comment]
This is now available as of v1.11, enjoy!
